src/viur/core/render/html/user.py

```diff
@@ -9,8 +11,8 @@ class Render(DefaultRender):  # Render user-data to xml
     verifySuccessTemplate = "user_verify_success"
     verifyFailedTemplate = "user_verify_failed"
     passwdRecoverInfoTemplate = "user_passwdrecover_info"
-    otp_add_template = "user_secondfactor_add"
-    otp_add_success_template = "user_secondfactor_add_success"
+    second_factor_add_template = "user_secondfactor_add"
```

Comment:
IMHO, the choice-template must be set also here, see https://github.com/viur-framework/viur-core/pull/845/files#r1320525611

Reply:
Yeah, that's done in #845. 

The PRs reflect just the consequence of my adjustments of the templates for the base. Therefore, the different templates are packed into different PRs.